ci-operator/step-registry/ipi/conf/gcp/ipi-conf-gcp-chain.yaml

```diff
@@ -4,6 +4,7 @@ chain:
   - ref: ipi-conf
   - ref: ipi-conf-telemetry
   - ref: ipi-conf-gcp
+  - ref: ipi-conf-gcp-osdisk-disktype
```

Comment:
@jianli-wei I have a concern here, one scenario will be missed: let installer choose disk type automatically.
Is it possible to add `ipi-conf-gcp-osdisk-disktype` to a few workflows instead?

Reply:
@yunjiang29 I'd like to confirm with you. You know, `controlPlane.platform.gcp.type` (along with `osDisk.diskType` and `osDisk.diskSizeGB`) & `compute.platform.gcp.type` would always be specified in step `ipi-conf-gcp` (see [here](https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/gcp/ipi-conf-gcp-commands.sh#L37-L60)). But, you are still concerned with specifying `osDisk.diskType` for compute/worker nodes? Thanks!

Reply:
Yes.
The reason for adding diskType for control plane nodes is due to a performance issue, OCPBUGS-1130, but I don't think changing the default value (`compute.platform.gcp.osDisk == null`) for a _feature_ is a good choice, as a subset of workflows are enough for testing purpose and others are not affected.

Reply:
Got it, let me update accordingly, thanks!

Reply:
@yunjiang29 I updated the ipi conf step, to skip setting disk type if the selected disk type is the default "pd-ssd". WDYT?